FAQ.md

```diff
@@ -88,3 +88,9 @@ If your client is <2.3.3, remove `tls-version-min 1.2` from your `/etc/openvpn/s
 Sysctl options are at `/etc/sysctl.d/20-openvpn.conf`
 
 ---
+
+**Q:** How can I access computers on the remote nework?
```

Comment:
Remote? or LAN?

Reply:
LAN is more clear,  I've changed it.

Reply:
I'm not sure why you call this remote ? From what I understand this allows you to connect to your LAN while being connected to the VPN

Reply:
Are you saying that if both your client LAN and the LAN behind the VPN server have the same subnet `192.168.1.0` and you access the router web page, the router that answers is the remote one and you want to exclude that subnet from VPN routing so that your local router answers instead?

From the original issue I understood the opposite and so you need to add a route to access the router on the remote LAN.

From @ruicruzpt comment:
> OpenVPN only allow me to surf public networks, not private network. Is there any config I need to add after using your script to allow me to add the 192.168.. range?

Reply:
For me, this isn't about "remote" LAN. OpenVPN will override the default route/gateway so all traffic will be routed to the OpenVPN server. You LAN will be inaccessible. `push "route 192.168.1.0 255.255.255.0"` will add another route on top of the new gateway so add traffic to your LAN will be forwarded to your local router and this be accessible again.

Reply:
But if openvpn overrides the default gateway then routing 192.168.1.0 to 255.255.255.0 redirects to the default gateway that is from openvpn. To properly exclude an IP the command is `route 192.168.1.0 255.255.255.0 net_gateway`.

With `route 192.168.1.0 255.255.255.0` we are simply advertising that the subnet 192.168.1.0 is available and should be routed through the vpn.
I don't host a vpn at home so I can't test. These are the links I read:
- https://openvpn.net/community-resources/expanding-the-scope-of-the-vpn-to-include-additional-machines-on-either-the-client-or-server-subnet/
- https://community.openvpn.net/openvpn/wiki/RoutedLans
- https://forums.openvpn.net/viewtopic.php?t=7065

Reply:
I'll take a look

Reply:
Ok, I had it the other way around, I just tested. Thanks!